lib/hets-rabbitmq-wrapper.rb

```diff
@@ -7,18 +7,41 @@ module HetsRabbitMQWrapper
   def self.run
```

Comment:
Also: Please rename the `run` method to `call`. This is more consistent with Ruby objects that can be called like a lambda, proc or a block. They all implement the `call` method.